src/lib/services/workflow-service.ts

```diff
@@ -114,9 +114,14 @@ export const fetchAllWorkflows = async (
   request = fetch,
   archived = false,
 ): Promise<CombinedWorkflowExecutionsResponse> => {
-  const query = decodeURIComponent(
-    parameters.query || toListWorkflowQuery(parameters, archived),
-  );
+  const rawQuery =
```

Comment:
Can we add some unit tests around the query in question?

Reply:
Yes, good call.